### PR TITLE
Assigment2 html

### PR DESCRIPTION
**Pull request by Althafkm**

### CODE REVIEW

**Comment by Althafkm** [comment]
Screenshot (118)

File: src/index.html [hunk 1]
```diff
@@ -6,12 +6,104 @@
...
+          </form>
+      </div>
+      <div class="box">
+        <a class="navbar" href="#home" >Home</a>
+        <a class="navbar" href="#news">News</a>
+        <a class="navbar" href="#contact">Contact</a>
+         <a class="navbar" href="#about">About</a>
+      </div>
+
+  </div>
```
Comment (AjayKomireddy) [inline src/index.html:46]: try to make it easier to read ,code seems fine.

**Comment by saikiransathpadi** [comment]
nav division item shoould be aligned center
images are not scaled properly, they should be scaled with the same width and height
always avoid using base64 image URLs its not a good practice
its a good practice to have alt text for image tags
will look good if the margin spacing are given for the divisions

**Comment by Althafkm** [comment]
Thank you for the clarification.

File: src/index.html (hunk 1 above)
Comment (Althafkm) [reply src/index.html]: yeah sure.

**Comment by Althafkm** [comment]
"always avoid using base64 image URLs its not a good practice" 
what else we have to use for this ?

**Pull request merged**